src/server/game/Movement/PathGenerator.cpp

```diff
@@ -245,19 +245,14 @@ void PathGenerator::BuildPolyPath(G3D::Vector3 const& startPos, G3D::Vector3 con
     // *** poly path generating logic ***
 
     // start and end are on same polygon
-    // just need to move in straight line
+    // handle this case like if they were 2 different polygons, building a line path split in few points
```

Comment:
(I know this is just a comment line, but it would be nice to be able to read it in correct English)
```suggestion
    // handle this case as if they were 2 different polygons, building a line path split in some few points
```